src/main/java/net/openhft/chronicle/queue/BufferMode.java

```diff
@@ -18,10 +18,26 @@
 
 package net.openhft.chronicle.queue;
 
+/**
+ * Enum representing the different buffer modes that can be used within Chronicle Queue.
+ * Each mode has a specific use case and behavior depending on the configuration.
+ */
 public enum BufferMode {
-    None,    // The default
+    /**
+     * The default buffer mode.
+     * No additional buffering or special handling is applied.
+     */
+    None,    // Default mode, no buffering
```

Comment:
Docs are doubled up here - you can consolidate to the `/** */` multi-line comments and remove the single line comments

Reply:
Reviewed to reduce comments that add no value -1200 lines